ui/nuxeo-filter.js

```diff
@@ -178,12 +180,38 @@ import { FiltersBehavior } from './nuxeo-filters-behavior.js';
     }
 
     _evaluate(document, user, expression) {
+      let res = false;
+
       try {
-        const fn = new Function(['document', 'user'], `return ${expression};`);
-        return fn.apply(this, [document, user]);
+        if (Nuxeo.UI && Nuxeo.UI.config && Nuxeo.UI.config.expressions && Nuxeo.UI.config.expressions.eval === false) {
```

Comment:
We could rename `eval` to `unsafeEval`

Reply:
Any comments on this?

Reply:
ah! sorry. I'd rather not flag the eval as unsafe as i don't think it was tbh. This is a choice about using it or not, that's all